modules/tests/shared/src/test/scala-3/io/circe/DerivesSuite.scala

```diff
@@ -137,4 +135,14 @@ class DerivesSuite extends CirceMunitSuite {
   checkAll("Codec[Foo]", CodecTests[Foo].codec)
   checkAll("Codec[RecursiveAdtExample]", CodecTests[RecursiveAdtExample].codec)
   checkAll("Codec[RecursiveWithOptionExample]", CodecTests[RecursiveWithOptionExample].codec)
+
+  property("Derives under `-Yexplicit-nulls` should work") {
+    val settings = staging.Compiler.Settings.make(compilerArgs = List("-Yexplicit-nulls"))
+    val explicitNullsCompiler = staging.Compiler.make(getClass.getClassLoader)(settings)
+    staging.run('{
+      val encoder = Encoder.AsObject.derived[Box[Wub]];
+      val decoder = Decoder.derived[Box[Wub]];
+      val codec = Codec.AsObject.derived[Box[Wub]];
+    })(using explicitNullsCompiler)
+  }
```

Comment:
Can you move this test to a jvm specific test suite (modules/tests/jvm/...`)?   I don't think this will work on ScalaJS or ScalaNative based on the errors I'm seeing in CI.

Reply:
OK, Done